### PR TITLE
added sentinel2-l2-analytics notebook version 1

### PR DESCRIPTION
**Pull request by Onyothi**
Added sentinel2-l2-analytics notebook version 1
https://up42.atlassian.net/browse/UP-5272
Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

**Review by marooned20 — APPROVED** [review]
Nice! great job!

File: examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-19 16:46:11,248 - Getting existing workflows in project ...\n",
+      "2021-03-19 16:46:11,718 - Got 1 workflows for project 1f2bad99-daad-4a60-9192-5169263caff6.\n",
+      "100%|██████████| 1/1 [00:00<00:00,  1.50it/s]\n",
+      "2021-03-19 16:46:12,396 - Using existing workflow: Sentinel2-l2-analytics-example - 00e94c32-7cfe-40b8-85b6-cc2ffa980c75\n",
+      "2021-03-19 16:46:16,343 - Added tasks to workflow: [{'name': 'esa-s2-l2a-gtiff:1', 'parentName': None, 'blockId': '4471e5ef-90f1-4bf0-9243-66bc9d8b4c99'}, {'name': 'sharpening:1', 'parentName': 'esa-s2-l2a-gtiff:1', 'blockId': 'e374ea64-dc3b-4500-bb4b-974260fb203e'}]\n"
```
Comment (thaisbendixen) [inline examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb:108]: This is unfortunately not the right block. Your workflow is called analytic example, hence you should use "esa-s2-l2a-gtiff-analytic" block. It's a bit confusing as there are several blocks on the platform right now with similar name

File: examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-19 16:45:59,477 - Authentication with UP42 successful!\n",
+      "2021-03-19 16:46:00,349 - Initialized Project(name: S2l2a-Analytics-data-example, project_id: 1f2bad99-daad-4a60-9192-5169263caff6, description: , createdAt: 2021-03-18T14:39:08.255254Z)\n"
```
Comment (thaisbendixen) [inline examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb:37]: Be sure to change analytic**s** to **analytic**. I made the same mistake when I created the block, but markus called to my attention that It doesn't mean the same thing. Analytics would mean that It has already been processed or analysed

**Review by markusuwe — COMMENTED** [review]
As Thais pointed out, unfortunately the wrong block was used.
Also, please add an introduction section introducing the dataset a bit and maybe an additional text describing the output.

File: examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb (hunk 1 above)
Comment (Onyothi) [reply examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb:108]: Thank you, I will begin to correct this.

**Review by chrieke — COMMENTED** [review]
LGTM, when using `bands=[3,2,1]`  in `job.plot_result()` it should look like true color.

**Review by rodrigoalmeida94 — COMMENTED** [review]
LGTM! As an optional suggestion, could you try to get rid of the UserWarnings? It looks a bit ugly if they are included.

**Review by rodrigoalmeida94 — APPROVED** [review]
LGTM!

File: examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb [hunk 3]
```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "competitive-bidding",
+   "metadata": {},
+   "source": [
+    "# Sentinel-2 L2A Analytic (GeoTIFF)\n",
+    "This notebook provides details on how to access Sentinel-2 data (L2A pre-processing or Bottom Of Atmosphere (BOA) reflectance images) with 13 bands. The 13 bands include 4 visible bands (10 m resolution), 6 Near-Infrared bands (20 m resolution), and 3 Short-Wave Infrared bands (60 m resolution).The Sentinel-2 mission is a land monitoring constellation of two satellites that provide high resolution optical imagery and provide continuity for the current SPOT and Landsat missions. The mission provides a global coverage of the Earth's land surface every 5 days, making the data of great use in on-going studies.Important application areas for Sentinel-2 imagery are: land cover monitoring (agriculture, forestry), coastal area monitoring, inland water monitoring, glacier monitoring and flood mapping\n",
+    "Sentinel-2 Level-2A products have been available over Europe since March 2018, and later extended gloablly as of December 2018."
```
Comment (thaisbendixen) [inline examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb:10]: From opening the jupyter notebook It seems there is a period missing.

**Review by thaisbendixen — APPROVED** [review]
LGTM! Mini suggestion: Would be nice to be consistent with the periods. Some of the comments and titles finish with a period some don't

**Pull request merged**